types/schemas/prototypeApplication/enums/FileType.ts

```diff
@@ -500,11 +500,12 @@ type WasteStorageDetails = 'wasteStorageDetails';
  */
 type WaterEnvironmentAssessment = 'waterEnvironmentAssessment';
 
+// @todo rename exported type when PrototypeApplication replaces Application
 /**
  * @id #FileType
  * @description Types of planning documents and drawings
  */
-export type FileType =
+export type PrototypeFileType =
```

Comment:
A few temporary "renames" required else we'll throw an error that more than one type with same name can't be exported !